### ISSUE TITLE
Document installation

### ISSUE THREAD

**Issue by bezda** [rusty-dev/jenkins-job-builder-pipeline#2]
Hi, could you please add some info how to integrate the plugin to JJB?

**Comment by rusty-dev** [comment]
Just install this module on the same machine that JJB is installed on. (You might want to use https://github.com/rusty-dev/jenkins-job-builder-pipeline/tree/stable until they merge latest changes to JJB)

**Comment by RudolfVonKrugstein** [comment]
What do you mean by "install this module on the same machine that JJB is installed on"?
How?

pip install jenkins-job-builder-pipeline

does not work.

**Comment by rusty-dev** [comment]
yeah, it's not on pypi, you can do `sudo pip install git+https://github.com/rusty-dev/jenkins-job-builder-pipeline.git@stable` or just clone the repo locally and run `setup.py` manually.
